internal/core/metadata/operators/addressable/db.go

```diff
@@ -5,6 +5,7 @@ import contract "github.com/edgexfoundry/go-mod-core-contracts/models"
 
 // AddressLoader provides functionality for obtaining Addressables.
 type AddressLoader interface {
+	AddAddressable(addressable contract.Addressable) (string, error)
```

Comment:
This needs to be in a different interface I believe, as it's not used "for obtaining Addressables."  I'd suggest creating an "AddressSaver" or something similar interface.

Reply:
Maybe `AddressAdder`